_maps/configs/limus.json

```diff
@@ -0,0 +1,37 @@
+{
+	"map_name": "Limus-class Xenobiology Cataloguing Vessel",
+	"map_short_name": "Limus-class",
```

Comment:
Please add a prefix and namelists here. Generic example below:

"prefix": "ISV",
"namelists": ["GENERAL", "SPACE", "NATURAL", "BEASTS"],